jingo_minify/tests.py

```diff
@@ -240,7 +243,7 @@ def test_static_override():
 def test_css(getmtime, time):
     getmtime.return_value = 1
     time.return_value = 1
-    env = jingo.env
+    env = jingo.get_env()
```

Comment:
The `.get_env()` requires at least jingo 0.8. We should probably add that to `setup.py` and update `requirements.txt`.